week-1-practise/level-1/2-syntax-error-2.js

```diff
@@ -1,8 +1,10 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a, b, c)
-return a + b + c;
+function addNumbers(a, b, c){
+
 
+return a + b + c;
```

Comment:
It is good to make sure that code inside the function is indented like this:
```suggestion
    return a + b + c;
```
When the code you are writing gets more complex, it will makes it much easier to read!